data/json/vehicle_parts.json

```diff
@@ -251,10 +251,19 @@
         "broken_symbol" : "#",
         "broken_color" : "brown",
         "durability" : 100,
+        "requirements": {
+          "install": {
+            "skills": [ [ "mechanics", 0 ] ],
```

Comment:
We should open an issue for that as a deficiency in the implementation I wrote - you should learn the required skills as opposed to just `mechanics`